client/src/pages/TakeSurvey/AlreadyCompletedMessage/AlreadyCompletedMessage.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+
+import { Typography } from '@material-ui/core'
+
+const AlreadyCompletedMessage = () => {
+  return <Typography variant='h3'>You have completed this survey! Thank you for your response.</Typography>
+
```

Comment:
Could just be an arrow function instead of using a return! ;D

Reply:
how the tables turn

Reply:
done